h/views/groups.py

```diff
@@ -124,6 +124,17 @@ def read_noslug(group, request):
     check_slug(group, request)
 
 
+@view_config(route_name='group_leave',
+             request_method='POST',
+             effective_principals=security.Authenticated)
+def leave(group, request):
+    """Route for leaving a group. Used by the Hypothesis client."""
```

Comment:
Can you please add another comment with a big FIXME that explains the real solution of providing a proper API endpoint?